persistent-sqlite/Database/Persist/Sqlite.hs

```diff
@@ -382,6 +382,7 @@ showSqlType :: SqlType -> Text
 showSqlType SqlString = "VARCHAR"
 showSqlType SqlInt32 = "INTEGER"
 showSqlType SqlInt64 = "INTEGER"
+showSqlType SqlWord64 = "INTEGER"
```

Comment:
I wonder if this is what's going on with the failing test you're finding. Based on [this](https://sqlite.org/datatype3.html) saying that `INTEGER` is (at most) an 8 byte size, I'd expect that `maxBound :: Word64` would be troublesome.

Reply:
Which, uh. Hm. Would suggest that the current behavior is Correct, since the test isn't failing right now, at least for SQLite.